.gitignore

```diff
@@ -109,6 +109,8 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+bin/
```

Comment:
This seems a strange thing to ignore as it doesn't exist and would typically contain things you wanted to keep in git...

Reply:
I was also a little less confident about this tbh. And since you pointed it out I just realised that I am probably creating virtual environment in wrong directory. I should be creating it in a subdirectory like ./venv which is already ignored. Reverted that change.
Thank you for pointing it out.